include/tvm/tir/var.h

```diff
@@ -103,6 +103,12 @@ class Var : public PrimExpr {
    * \param span The location of this object in the source code.
    */
   TVM_DLL explicit Var(String name_hint, Type type_annotation, Span span = Span());
+  /*!
+   * \brief Make a new copy of var with same type, but a different nam
+   * \param name The new name to be used.
+   * \return the new Var copy
+   */
+  TVM_DLL Var copy_with_name(const String& name) const;
```

Comment:
CamelCase

Reply:
it follows the cnovention existing methods like `Var::copy_with_dtype`